nexus-repository-helm/pom.xml

```diff
@@ -24,7 +24,6 @@
 
```

Comment:
See `<version>` tag above this (~line 22). To keep continuity with prior releases, and avoid "surprise", I think we should make this version 0.0.13 (instead of 1.0.0), which will become 0.0.14 at the next release.